app/common/lib/historyUtil.js

```diff
@@ -144,6 +145,19 @@ const getKey = (siteDetail) => {
   return null
 }
 
+const isHistorySite = (site) => {
+  if (!site) {
+    return false
+  }
+  const order = site.get('order')
+  const location = site.get('location')
+  const tags = site.get('tags')
```

Comment:
as an alternative we can add `type` property to the history object like we have it for bookmarks and bookmarks folders